crates/libs/link/src/lib.rs

```diff
@@ -5,6 +5,9 @@
 #[cfg(all(windows, target_arch = "x86"))]
 #[macro_export]
 macro_rules! link {
+    ($library:literal "cdecl" $($link_name:literal)? fn $($function:tt)*) => (
+        $crate::link!($library "C" $($link_name)? fn $($function)*);
+    );
```

Comment:
I still don't understand which problem this solves.

Reply:
Well, @ChrisDenton suggests that there're still some old code (authored by other users?) uses "cdecl". This is for compatibility because "cdecl" raises a warning and will become an error soon.

Reply:
Okay so it is about the lint, thanks. That seems worth a comment.
```suggestion
    // Avoid future-compat lint if bindings are still using the old "cdecl" form of the bindings.
    ($library:literal "cdecl" $($link_name:literal)? fn $($function:tt)*) => (
        $crate::link!($library "C" $($link_name)? fn $($function)*);
    );
```

Reply:
Commends added.